### ISSUE TITLE
Sound not working on both monitors?

### ISSUE THREAD

**Issue by wldkodiak** [MonitorControl/MonitorControl#1118]
### Before opening the issue, have you...?

- [X] Searched for existing issues
- [X] Read through [the Monitor-Troubleshooting Wiki](https://github.com/MonitorControl/MonitorControl/wiki/Monitor-Troubleshooting)
- [X] Updated MonitorControl to the latest version (if applicable)

### Describe the issue

MBP 2021 M1 Max
I have volume and dimming controls registering on screen for both monitors.
Dimming works on both.
However only sound is coming out of one monitor....

I have the updated app and both screens are brand new LG HDR 4k 32"
Also noteworthy - I am running parallels on the right monitor so I can have windows.
The monitor on the left work perfectly but I am just trying to get sound to come out of the one on the right.

Thanks in advance!

### Expected behavior

Need sound coming out of both monitors

### Anything else?

[Screen Shot 2022-07-24 at 4.21.03 PM.PDF](https://github.com/MonitorControl/MonitorControl/files/9176606/Screen.Shot.2022-07-24.at.4.21.03.PM.PDF)
[Screen Shot 2022-07-24 at 4.20.30 PM.PDF](https://github.com/MonitorControl/MonitorControl/files/9176607/Screen.Shot.2022-07-24.at.4.20.30.PM.PDF)

### Environment Information (please complete the following information)

```markdown
- macOS version:
- Mac model:
- MonitorControl version:
- Monitor(s):
- Monitor Cable(s)/Connection(s):
- Apple Silicon/M1 (yes or no):
```

**Comment by wldkodiak** [comment]
OK I FOUND OUT WHAT NEEDED TO HAPPEN....

So on mac I needed to open the app from apple installed on all macs. "Audio MIDI"

In there make a multi-output device and add both monitors to that.

Then right click on it and select use as output device. 

BOOM audio in both monitors working perfectly.

Here is the video I used to fix the issue.

[https://www.youtube.com/watch?v=EgTx2xMTDNM]

thanks!

**Comment by wldkodiak** [comment]
AND ONE LAST THING........

After setting in on the mac system side I had to go to the settings in parallels and select the same output setup.

When i was playing something on the mac side it would us both monitors then when i played something on the windows side it would only use that one monitor. 

So I am good now everything is functioning perfectly!

**Comment by waydabber** [comment]
Yes, macOS normally uses only one audio output but you can create multi-output devices. Great solution!

I'll move this over to discussion so others might be edified!
